EMF/events/family_events.txt

```diff
@@ -935,65 +935,68 @@ character_event = {
 ##############################################################
 
 # First event of the series
+
 character_event = {
-	id = 37050
-	desc = EVTDESC37050
-	picture = GFX_evt_marriage
+	id = 37049
 	
-	min_age = 16
+	hide_window = yes
+	
+	ai = no
 	capable_only = yes
 	prisoner = no
+	min_age = 16 # EMF
 	
 	trigger = {
-		is_ruler = no
-		NOT = {
-			trait = bastard
-		}
-		has_ambition = obj_get_married
-		NOT = {
-			has_minor_title = title_regent
-		}
-		is_betrothed = no
-		NOT = {
-			any_consort = {
-				always = yes
-			}
-		}
-		
-		liege = {
-			religion_group = christian
-			is_close_relative = ROOT
-		}
-		
-		NOT = { has_character_flag = flag_denied_marriage }
-		NOT = { has_character_flag = flag_promised_marriage }
+		religion_group = christian
 	}
 	
 	mean_time_to_happen = {
-		months = 48 # EMF: Was 24 in vanilla
-		
-		modifier = {
-			factor = 0.25
-			is_female = yes
-			trait = lustful
-		}
-		modifier = {
-			factor = 0.5
-			trait = ambitious
-		}
-		modifier = {
-			factor = 2.0
-			trait = slothful
-		}
-		modifier = {
-			factor = 5.0
-			trait = content
+		48 # EMF: Was 24 in vanilla
```

Comment:
Here on line 954 of family_events.txt, it looks like you mean `months = 48` rather than just `48`.